index.html

```diff
@@ -5,61 +5,170 @@
+                            my work and experience!
+                        If you like what you see and have a
+                            project you need coded,  don’t hestiate
+                            to contact me.
+                    </p> 
+                                          
+                </section>
+
                 
-            <h1>Laura Jong</h1>
-            <h2>Software Developer</h2>
-            <p>I can help you build a product , feature or website Look through some of my work and experience!
-                 If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
-        </div>
-    </section>
-    
+            
+        </section>
 
+        <section> <hr></section>
 
 
+    <!-- Middle Section -->
+
+    <section class="skills-section">
+
    
+
+        <section class="ellipse-2"></section>
+    
+        <ul class="language-list-1">
+            <li class="col-header"><h3> Language </h3></li>
+            <li class="list-items">Javascript</li>
+            <li class="list-items">Ruby</li>
+            <li class="list-items">Html</li>
+            <li class="list-items">CSS</li>    
+        </ul>
+
+        <ul class="language-list-2">
+            <li class="col-header"><h3>Frameworks</h3></li>
+            <li class="list-items">Bootstrap</li>
+            <li class="list-items">Ruby on Rails</li>
+            <li class="list-items">RSPec</li>
+            <li class="list-items">CapyBara</li>
+            <li class="list-items">Selenium</li>
+            
+        </ul>
+
+        <ul class="language-list-3">
+            <li class="col-header"><h3>Skilss </h3></li>
+            <li class="list-items">Codekit</li>
+            <li class="list-items">Guthub</li>
+            <li class="list-items">Codepen</li>
+            <li class="list-items">Gitlab</li>
+            <li class="list-items">Terminal</li>
+            
+        </ul>
+    </section>
+
+    <div class="icons-section">
+
     
+    <section> 
+
+    <img src="images/git.svg" alt="">
+    <img src="images/linkedin.svg" alt="">
+    <img src="images/othericon.svg" alt="">
+    <img src="images/twitter.svg" alt="">
+    <img src="images/micon.svg" alt="">
+    </section>
+    <section class="arrow-bottom">
+        <img class="arrow-bottom" src="images/elementright.svg" alt="">
+    </section>
+    
+   </div>
+
+
+   <section class="my-work">
+    <p>MY RECENT WORK</p>
+
+    <hr>
+</section > 
+
+<section class="post-container">
+<section class="white-bg"> </section>
+<section class="posts">
+    <h2>Multi-Post Stories</h2>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+         when an unknown printer took a standard dummy text.</p>
+
+    <P class="tags">css <span class="span"></span>  html <span class="span"></span>bootstrap <span class="span"></span> Ruby </p>
```

Comment:
- [ ] The list of project technologies are a list ul and each of them a li, please kindly fix it🙃 🙃